docs/QA.md

```
@@ -46,10 +46,10 @@ Test types
 
 Run tests:
 
-    - run all tests: `npm run-script test:nightly`
-    - run all tests from particular area: `npm run-script test:it ./test/functional/integration/challenge/`
-    - run all tests for a test file: `npm run-script test:it ./test/functional/integration/challenge/query-challenge-data.it-spec.ts`
-    - run e2e tests with coverage: `npm run test:e2e-cov`
+    - run all tests: `ppnpm run test:nightly`
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)